app/src/pages/OnDeviceDisplay/RunningProtocol.tsx

```diff
@@ -79,7 +81,7 @@ export function RunningProtocol(): JSX.Element {
   const currentRunCommandIndex = robotSideAnalysis?.commands.findIndex(
     c => c.key === currentRunCommandKey
   )
-  const runStatus = useRunStatus(runId)
+  const runStatus = useRunStatus(runId, undefined, RUN_STATUS_REFETCH_INTERVAL)
```

Comment:
I think passing as an object would be better than passing `undefined`

```suggestion
    const runStatus = useRunStatus(runId, {
    refetchInterval: RUN_STATUS_REFETCH_INTERVAL,
  })
```

Reply:
yes good call. also changing it to 5 seconds!